free-2.md

```diff
@@ -11,41 +11,106 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would allow people to have more control over how they buy and sell
+shares.  Currently, most people invest in the market over long-term periods.
+Alternatively, they give their money to a hedge fund, who will buy shares for
+them.  These hedge funds use complicated algorithms to determine at what price to
+sell a stock, and how long it should be available for.  However, these techniques
+or not available to typical investors.  Also, the code is typically written in C
+for efficiency.  A DSL for the Stock Market would help people who want to
+precisely control the timing and price of their shares but do not want to give
+their money to a hedge fund.  With this DSL they could create functions that react
+to the market to determine the best investment strategy.
```

Comment:
I've seen a few of these ideas, and they make me a bit nervous in that they're looking to solve a very complex problem in a very simple way. While I believe that you could design a DSL that makes it easier to buy and trade stocks, and that it could be accessible, I worry that it would obscure too many important details. I wonder if the domain is right here; should non-technical and non-experienced people necessarily have access to such a tool?